src/components/FavoritesBtn.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+import { FavoriteMovie } from '../models/index';
+import api from '../service/api';
+
+import '../styles/FavoritesBtn.scss';
+
+const FavoritesBtn: React.FC<FavoriteMovie> = ({ movieId }) => {
+  const [icon, setIcon] = useState(api.isIdInFavorites(movieId));
+
+  const heartUnfilledIcon = '/images/favoriteBtn/heartFilled.svg';
+  const heartFilledIcon = '/images/favoriteBtn/heartUnfilled.svg';
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
+    event.preventDefault();
+    api.isIdInFavorites(movieId)
```

Comment:
why not use icon?

Reply:
You're right. I have not noticed it at the beginning. I am going to use a state variable here